deployment/cloud/aws/infrastructure/modules/server/userdata

```diff
@@ -41,9 +41,11 @@ runcmd:
   # install pixi
   - echo "Installing Pixi"
   - export PIXI_HOME=/opt/pixi
+  - export HOME=/root
```

Comment:
[this line](https://github.com/prefix-dev/pixi/blob/475ad6149cc01d6fdd507857dca58d500e1a0785/install/install.sh#L9) of Pixi's install script was failing to execute on the server without this, bc usually `$HOME` is not set during userdata/cloud-init execution. Set it while installing so Pixi could actually run